### ISSUE TITLE
[Feature Request] Prompt area resize option (and maybe scroll bar).

### ISSUE THREAD

**Issue by xEvilVeganx** [Acly/krita-ai-diffusion#1471]
AMAZING plugin, but I'm getting frustrated with something pretty minor.  I'm doing a live mode inpainting and the prompt window is tiny compared to everything else and doesn't change when I adjust the # of lines or the docker area, so it's hard to find the part of the prompt I want to edit.

![image](https://github.com/user-attachments/assets/3dc5591b-4975-44d4-a3f3-5678d7249a61)

Ideally when I try to resize the docker I'd get more/fewer lines based on overall percentage of space available.  But it doesn't seem to matter:
![image](https://github.com/user-attachments/assets/c8e189a9-9ea8-4c33-a4ef-e31185ac2d3a)

If I could resize the text area with a little corner drag down and/or it added a scroll bar when the text extended past the current viewing area (like I have in this current browser window on GitHub), that would be really cool:

![image](https://github.com/user-attachments/assets/a11b2e79-9b46-40d2-9699-a7c3480430f5)

**Comment by xEvilVeganx** [comment]
Oh and also it would be nice to be able to adjust the preview image size to whatever we want, or scale it with docker size.

**Comment by austeregrim** [comment]
![image](https://github.com/user-attachments/assets/4f4af9fb-7fb0-4733-8a25-54152963477c)
You can change the "prompt line count" in the settings. Although not dynamic resize like you're asking, you can set it a bit higher and resolve your issue.

**Comment by xEvilVeganx** [comment]
>  You can change the "prompt line count" in the settings. Although not dynamic resize like you're asking, you can set it a bit higher and resolve your issue.

That's my problem, it doesn't change anything.   It works in generate mode, but not live mode.  If I set it to 1 it has 1 line, if I set it to 2+ it gives me roughly 3.5 lines and doesn't change as I go up to 10, and turning on the negative prompt makes it a little worse.

![image](https://github.com/user-attachments/assets/f59d4a5f-d80a-40e1-9038-d9229b169e24)

![image](https://github.com/user-attachments/assets/d92b0b6a-bc23-413f-842a-532c7c4010e1)

![image](https://github.com/user-attachments/assets/213a5325-a6bd-4c14-bfc8-8d625584bdcd)

**Comment by austeregrim** [comment]
Yes, that looks like a bug, not a feature request.

**Comment by xEvilVeganx** [comment]
Good point.  Either way, it would be better if I could just grab the corner and floop it bigger or smaller as needed.  Make it change size relative to how big the docker area is.

**Comment by Acly** [comment]
Prompt boxes are now drag-resizeable in [v1.31.0](https://github.com/Acly/krita-ai-diffusion/releases/tag/v1.31.0)
